source/main.py

```diff
@@ -76,34 +76,40 @@ def synchronous_with_tracker():
         - does use the tracker
         """
 
-        # by defaul tracker needs model to be run
-        tracker_found_bounding_box = False
+    
+        # model needs to run on first iteration
+        best_bounding_box = None
+
         for counter in count(start=0, step=1): # counts up infinitely starting at 0
+            # runs for the first 450 frames of video since there is a faulty frame later on
+            if counter == 450:
+                break
             frame = get_latest_frame()
 
-            if counter % 20 == 0 or not tracker_found_bounding_box:
-                #
-                # call model
-                #
-                boxes, confidences, classIDs = model(frame, confidence, threshold)
-                tracker_found_bounding_box = tracker.init(frame,boxes)
-                # FIXME: best_bounding_box needs to be calculated here! (either call tracker)
+            # run model every 40 frames or whenever the tracker fails
+            if counter % 40 == 0 or (best_bounding_box is None):
```

Comment:
Could you put the `40` limit inside the info.yaml parameters, and then import it? I know the 20 wasn't in there, but it probably should've been